samples/data/form/overview/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
As per Martin's request we need to rename the data forlder to something else. He suggested "inputs".